plugin/handlergen.go

```diff
@@ -781,10 +781,73 @@ func (p *OrmPlugin) isFieldOrmable(message *generator.Descriptor, fieldName stri
 	return ok
 }
 
-func (p *OrmPlugin) removeChildAssociations(message *generator.Descriptor) {
+func (p *OrmPlugin) handleChildAssociations(message *generator.Descriptor) {
 	ormable := p.getOrmable(p.TypeName(message))
 	for _, fieldName := range p.getSortedFieldNames(ormable.Fields) {
-		p.removeChildAssociationsByName(message, fieldName)
+		p.handleChildAssociationsByName(message, fieldName)
+	}
+}
+
+func (p *OrmPlugin) handleChildAssociationsByName(message *generator.Descriptor, fieldName string) {
+	ormable := p.getOrmable(p.TypeName(message))
+	field := ormable.Fields[fieldName]
+
+	if field == nil {
+		return
+	}
+
+	if field.GetHasMany() != nil || field.GetHasOne() != nil || field.GetManyToMany() != nil {
+		var assocHandler string
+		switch {
+		case field.GetHasMany() != nil:
+			switch {
+			case field.GetHasMany().GetAppend():
+				assocHandler = "Append"
+			case field.GetHasMany().GetClear():
+				assocHandler = "Clear"
```

Comment:
```suggestion
			case field.GetHasMany().GetClear():
				assocHandler = "Clear"
			case field.GetHasMany().GetAppend():
				assocHandler = "Append"
```
For consistency with the other switches, it should be in this order